js/main.js

```diff
@@ -21,8 +21,100 @@ $entryForm.addEventListener('submit', function () {
   entryData.photoUrl = $entryForm.elements.photoUrl.value;
   entryData.notes = $entryForm.elements.notes.value;
   entryData.entryId = data.nextEntryId;
-  data.entries.push(entryData);
+  data.entries.unshift(entryData);
   data.nextEntryId++;
+  $newEntries.prepend(renderData(entryData));
   $picView.setAttribute('src', 'images/placeholder-image-square.jpg');
   $entryForm.reset();
+
 });
+
+function entryDisplay(event) {
+  for (var i = 0; i < data.entries.length; i++) {
+    var $newEntry = renderData(data.entries[i]);
+    $newEntries.appendChild($newEntry);
+  }
+  for (var j = 0; j < $view.length; j++) {
+    if ($view[j].getAttribute('data-view') === data.view) {
+      $view[j].className = ' view ';
+    } else {
+      $view[j].className = 'view hidden';
+    }
+  }
+}
+
+document.addEventListener('DOMContentLoaded', entryDisplay);
+
+// <li class="row">
+//  <div class=" column-half">
+//    <div class="picHolder"><img class="picView" src="images/placeholder-image-square.jpg"></div>
+//  </div>
+//  <div class="column-half entryInfo">
+//    <h2 class="entryTitle" >Marvels Movie</h2>
+//    <p class="entryNote">Our salute to Captain America and his uniforms in the MCU. Which is your favorite? Complete your Marvel Studios' Captain
+//      America collection on Digital now!</p>
+//  </div>
+// </li>
+
+var $newEntries = document.querySelector('.list-group');
+
+function renderData(data) {
+  var $newList = document.createElement('li');
+  $newList.setAttribute('class', 'row');
+
+  var $listPic = document.createElement('div');
+  $listPic.setAttribute('class', 'column-half');
+
+  var $picHolder = document.createElement('div');
+  $picHolder.setAttribute('class', 'picHolder');
+
+  var $pic = document.createElement('img');
+  $pic.setAttribute('class', 'picView');
+  $pic.setAttribute('src', data.photoUrl);
+
+  var $entryInfo = document.createElement('div');
+  $entryInfo.setAttribute('class', 'column-half entryInfo');
+
+  var $entryTitle = document.createElement('h2');
+  $entryTitle.textContent = data.title;
+
+  var $entryNote = document.createElement('p');
+  $entryNote.textContent = data.notes;
+
+  $newList.appendChild($listPic);
+  $newList.appendChild($entryInfo);
+
+  $listPic.appendChild($picHolder);
+  $picHolder.appendChild($pic);
+
+  $entryInfo.appendChild($entryTitle);
+  $entryInfo.appendChild($entryNote);
+
+  return $newList;
+}
+
+var $tabList = document.querySelector('.tabList');
+var $view = document.querySelectorAll('.view');
+var $newButton = document.querySelector('.newButton');
+var $submitButton = document.querySelector('.submitButton');
+
+function viewchange(event) {
```

Comment:
Instead of making this function work off of the `event` object, you should make it accept an argument of a string containing the name of the target view. That way you can reuse this function any time you want to move between views anywhere in your code.